feature/home/src/main/java/com/linku/home/ui/alarm/component/listcontent/AlarmAppendStateFooter.kt

```
@@ -0,0 +1,98 @@
+            appendState = LoadState.Loading,
+            onRetry = {}
+        )
+    }
+}
+
+@Preview(showBackground = true)
+@Composable
+private fun AlarmAppendStateFooterErrorPreview() {
+    LinkuPreview {
+        AlarmAppendStateFooterContent(
+            appendState = LoadState.Error(Exception()),
```

detekt (1.23.8)

[warning] A call to the default constructor of an exception was detected. Instead one of the constructor overloads should be called. This allows to provide more meaningful exceptions.

(detekt.exceptions.ThrowingExceptionsWithoutMessageOrCause)